cmd/output/error.go

```diff
@@ -76,3 +80,54 @@ func PrintErrorInvalidOrganizationSlug(slug string) {
 func PrintErrorInvalidAppSlug(appSlug string) {
 	PrintError("Invalid app %q.", "Must contain only lower-case alphanumerical characters and dashes.", appSlug)
 }
+
+func PrintErrorAppNotFound(ai appident.AppIdentifier) {
+	PrintError(
+		"App not found",
+		"The app you are trying to delete \"%s/%s\" cannot be found. Did you specify the correct organization and app slug?",
```

Comment:
The naming of the function makes it seem like it was intended to be generic and used multiple places 😄 
So we should replace the word "delete" with something generic like "access"/"manage"  or just  remove "you are trying to delete"